rust/operator-binary/src/backend/tls.rs

```diff
@@ -36,6 +38,23 @@ use super::{
     SecretBackend, SecretBackendError, SecretContents,
 };
 
+/// As the Pods will be evicted [`EXPIRE_PODS_BEFORE_CERT_EXPIRES_PERIOD`] before
+/// the cert actually expires, this results in a restart in approx every 2 weeks,
+/// which matches the rolling re-deploy of k8s nodes of e.g.:
+/// * 1 week for IONOS
+/// * 2 weeks for some on-prem k8s clusters
+pub const DEFAULT_MAX_CERT_LIFETIME: std::time::Duration =
+    std::time::Duration::from_secs(15 * 24 * 60 * 60);
+/// This needs to be sufficiently larger than [`EXPIRE_PODS_BEFORE_CERT_EXPIRES_PERIOD`], so that the
+/// Pod does not immediately gets evicted once it's created.
+pub const MIN_MAX_CERT_LIFETIME: Duration = Duration::days(1);
```

Comment:
This seems massive for a minimum.

Reply:
The whole minimum constraint for the maxCertLifetime was removed

Reply:
Implementation switched to what we have decided upon in the Arch meeting